gradle/libs.versions.toml

```
@@ -16,6 +19,10 @@ pluginPublish = { id = "com.gradle.plugin-publish", version.ref = "pluginPublish
 spotless = { id = "com.diffplug.spotless", version.ref = "spotless" }
 
 [libraries]
+# runtime
+jacksonModuleKotlin = { module = "com.fasterxml.jackson.module:jackson-module-kotlin", version.ref = "jackson" }
+jacksonDataformatYaml = { module = "com.fasterxml.jackson.dataformat:jackson-dataformat-yaml", version.ref = "jackson" }
```

nit: alphabetical order

```suggestion
jacksonDataformatYaml = { module = "com.fasterxml.jackson.dataformat:jackson-dataformat-yaml", version.ref = "jackson" }
jacksonModuleKotlin = { module = "com.fasterxml.jackson.module:jackson-module-kotlin", version.ref = "jackson" }
```